mne/viz/backends/renderer.py

```diff
@@ -37,10 +37,17 @@ def _reload_backend(backend_name):
 
 
 def _get_renderer(*args, **kwargs):
-    set_3d_backend(_get_3d_backend(), verbose=False)
+    _get_3d_backend()
```

Comment:
I don't see why we'd need to set the get when the get does a `_reload_backend` under the hood already?

Reply:
heh, this might explain why I sometimes see `setting pyvista 3d backend` twice in a row when running some of my scripts.